csds-client/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"envoy-tools/csds-client/client"
+	"fmt"
+)
+
+func main() {
+	_, error := client.New()
+	if error != nil {
+		fmt.Println(fmt.Errorf("%v", error).Error())
```

Comment:
You can just do "fmt.Printf("%v\n", error)

Reply:
Solved.